CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
-Next Release
--------------
+2.0.0
```

Comment:
I think, instead of maintaining a changelog, it'd be nice to leverage [Github releases](https://help.github.com/articles/creating-releases/).

I like how Bootstrap handles it: https://github.com/twbs/bootstrap/blob/master/CHANGELOG.md